defaults/main.yml

```diff
@@ -6,3 +6,4 @@ nvm_install_path: "{{ nvm_home }}/.nvm"
 nvm_node_version: "v6.0.0"
 nvm_node_dir: "{{ nvm_install_path }}/versions/node/{{ nvm_node_version }}"
 nvm_npm_path: "{{ nvm_node_dir }}/bin/npm"
+nvm_version: v0.34.1
```

Comment:
Are there going to be any negative implications for setups that have used this role in the past updating to nvm 0.34.1, from 0.32.1?